README.md

```diff
@@ -67,6 +67,69 @@ And defines the following commands, mostly a mapping to xmonad key combinations.
 * `mod1 + i` - display the current layout for each screen
 * `mod1 + z` - force windows to be reevalulated
 
+Available Layouts
+-----------------
+
+Amethyst allows you to cycle among several different window layouts.
+Layouts can also be enabled/disabled to control whether they appear in the cycle sequence at all.
+
+* *Tall*
+> The default layout. This gives you one "main pane" on the left, and one other
+> pane on the right. By default, one window is placed in the main pane
+> (extending the full height of the screen), and all remaining windows are
+> placed in the other pane. If either pane has more than one window, that pane
+> will be evenly split, horizontally, to show them all.
```

Comment:
I think vertically is more correct than horizontally as it is partitioned along the vertical axis.

Reply:
Hmm, yes, this is confusing.  To me "horizontally" is more intuitive as the layout within the pane resembles the "Row" layout.  Seems readers may disagree on which terminology is more intuitive, so I'll try to be even more explicit to avoid confusion.  This comment also affects the descriptions of some of the other layouts as well.

Reply:
Maybe rather than "split vertically" or "split along its vertical axis", it's better to just stick with "split into rows".